server/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name    = "svc-cargo"
+version = "0.1.0"
+edition = "2021"      # Allowed values are 2015, 2018, 2021
+# Rust Editions: https://doc.rust-lang.org/edition-guide/index.html
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+vendored-openssl = ['openssl/vendored']
+
+[dependencies]
+axum                      = "0.5"
+cargo-husky               = "1"
+chrono                    = { version = "0.4", features = ["serde"] }
+env_logger                = "0.9"
+futures                   = "0.3"
+hyper                     = "0.14"
+log                       = "0.4"
+openssl                   = "0.10"
+prost                     = "0.11"
+prost-build               = "0.11"
+prost-types               = "0.11"
+serde                     = "1.0"
+serde_json                = "1.0"
+svc-pricing-client-grpc   = { git = "https://github.com/Arrow-air/svc-pricing" }
+svc-scheduler-client-grpc = { git = "https://github.com/Arrow-air/svc-scheduler" }
+svc-storage-client-grpc   = { git = "https://github.com/Arrow-air/svc-storage" }
```

Comment:
Will all need to specifically point to the main branch once these services are all released.
So we can't merge before that.

Reply:
Updated, will rerun CI when svc-scheduler gets merged in